api/wild/index.js

```diff
@@ -0,0 +1,75 @@
+import { renderStatsCard } from "../../src/cards/stats-card.js";
+import {
+  clampValue,
+  CONSTANTS,
+  parseArray,
+  parseBoolean,
+  renderError,
+} from "../../src/common/utils.js";
+
+export default async (req, res) => {
+  const {
+    hide,
+    hide_title,
+    hide_border,
+    card_width,
+    hide_rank,
+    show_icons,
+    line_height,
+    title_color,
+    ring_color,
+    icon_color,
+    text_color,
+    text_bold,
+    bg_color,
+    theme,
+    disable_animations,
+    border_radius,
+    border_color,
+  } = req.query;
+  res.setHeader("Content-Type", "image/svg+xml");
+
+  try {
+    const stats = {
+      totalStars: req.query.total_stars,
```

Comment:
I don't think giving users the ability to construct their own `stats` and `language` cards are wise. I think the idea of having an endpoint where people can create custom cards (i.e. `wild`) is very cool. I would, however, make the stats and language card items reserved and throw an error if people try to use those.